jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-jruby9k-json/Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+node {
```

Comment:
We could just call this one latest-jruby9k, I think, and not work "json" into the name since we'll probably just stick with json always for Puppet 5-based runs.